src/rust/zh.manhuagui/res/source.json

```
@@ -3,7 +3,7 @@
 		"id": "zh.manhuagui",
 		"lang": "zh",
 		"name": "ManHuaGui",
-		"version": 2,
+		"version": 3,
 		"url": "https://www.manhuagui.com/",
```

should change this while we're at it
```suggestion
		"url": "https://www.manhuagui.com",
```

---

My computer has been turned off and can't be handled for the time being.

---

done